tests/testthat/test-analyse.R

```diff
@@ -48,6 +48,26 @@ test_that("basic constructions of `analysis` work as expected",{
     )
     expect_true(validate(x))
 
+
+    x <- as_analysis(
+        results = list(
+            list(p1 = list("est" = 1, "df" = 4, "se" = NA)),
+            list(p1 = list("est" = 2, "df" = 3, "se" = NA))
+        ),
+        method = method_bayes(n_samples = 2)
```

Comment:
Should this not error due to the different DF  ?

Reply:
I think we are not verifying that all the df are the same in `validate.rubin` of the analysis output.